clj/src/slipstream/ui/models/dashboard.clj

```diff
@@ -17,17 +18,35 @@
   [cloud-usages]
   (cons
     {:cloud         (t :all-clouds.title)
+     :global-usage? true
      :current-usage (->> cloud-usages (map :current-usage) (reduce +))
      :quota         (->> cloud-usages (map :quota)         (reduce +))}
     cloud-usages))
 
+(defn- visible?
+  [cloud-usage]
+  (let [configured-clouds (into #{} (current-user/configuration :configured-clouds))]
+    (or
+      (-> cloud-usage :global-usage?)
+      (-> cloud-usage :current-usage pos?)
+      (-> cloud-usage :cloud configured-clouds))))
```

Comment:
We could avoid systematic check with configured-clouds like that?
(defn- visible?
    [cloud-usage]
    (or
      (-> cloud-usage :global-usage?)
      (-> cloud-usage :current-usage pos?)
      (-> cloud-usage :cloud (set (current-user/configuration :configured-clouds)))))

Proposition less readable and not tested.